src/InstrumentationEngine.Attach/EngineConfiguration.xsd

```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <!-- root type -->
+    <xs:element name="InstrumentationEngineConfiguration">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="InstrumentationEngine">
+                    <xs:complexType>
+                        <xs:sequence>
+                            <xs:element name="Settings" type="settingsType" />
+                        </xs:sequence>
+                    </xs:complexType>
+                </xs:element>
+                <xs:element name="InstrumentationMethods" type="instrumentationMethodsType" />
+            </xs:sequence>
+        </xs:complexType>
+    </xs:element>
+
+    <!-- level 1 types -->
+    <xs:complexType name="instrumentationMethodsType">
```

Comment:
>instrumentationMethodsType [](start = 26, length = 26)

nit: Capitalize the type names so that they generate the properly cased class names.